Description: Upstream changes introduced in version 0.42-2

 Add an 'install' target to the Makefile

Author: Javier Fernandez-Sanguino Pen~a <jfs@debian.org>
Forwarded: no
Origin: vendor

--- a/Makefile
+++ b/Makefile
@@ -1,3 +1,13 @@
+#
+# Makefile for sshuttle
+
+SSHUTTLE_DIR=/usr/lib/sshuttle
+BIN_DIR=/usr/sbin/
+
+SSHUTTLE_MAIN=main.py
+SSHUTTLE_SCRIPTS=assembler.py firewall.py hostwatch.py options.py ssh.py \
+	client.py helpers.py server.py ssnet.py
+
 PANDOC:=$(shell \
 	if pandoc </dev/null 2>/dev/null; then \
 		echo pandoc; \
@@ -15,5 +25,16 @@
 %.8: %.md
 	$(PANDOC) -s -r markdown -w man -o $@ $<
 
+install: install-compat
+	[ -d "$(PREFIX)$(SSHUTTLE_DIR)" ] || mkdir -p $(PREFIX)$(SSHUTTLE_DIR)
+	install -m755 $(SSHUTTLE_MAIN) $(PREFIX)$(SSHUTTLE_DIR)
+	install -m644 $(SSHUTTLE_SCRIPTS) $(PREFIX)$(SSHUTTLE_DIR)
+	[ -d "$(PREFIX)$(BIN_DIR)" ] || mkdir -p $(PREFIX)$(BIN_DIR)
+	( cd  $(PREFIX)$(BIN_DIR) && ln -s $(SSHUTTLE_DIR)/$(SSHUTTLE_MAIN) sshuttle )
+
+install-compat:
+	[ -d "$(PREFIX)$(SSHUTTLE_DIR)/compat" ] || mkdir -p $(PREFIX)$(SSHUTTLE_DIR)/compat
+	install -m644 compat/*.py $(PREFIX)$(SSHUTTLE_DIR)/compat
+
 clean:
 	rm -f *~ */*~ .*~ */.*~ *.8 *.tmp */*.tmp *.pyc */*.pyc
